astropy/units/tests/test_quantity_ufuncs.py

```diff
@@ -52,6 +56,21 @@ def test_testwarn(tw):
         tw.f(*tw.q_in)
 
 
+@pytest.fixture
+def fast_thread_switching():
+    """Fixture that reduces thread switching interval.
+
+    This makes it easier to provoke race conditions.
+
+    TODO: this is copied from another branch which adds it elsewhere. It
```

Comment:
Well... Should we work to get that other PR merged first and then rebase this one on top of upstream `master` after that is merged? If so, please turn this PR into draft status and put a note on the top post to indicate that this PR is dependent on that other PR. (GitHub still has no proper linking like JIRA.) 

But that said, is a new fixture really necessary? I can we use `setup_class` and `teardown_class` instead within `TestUfuncHelpers`?

Reply:
#11224 is merged, so this is moot now. Please rebase and use your fixture from `conftest.py`. Thanks! 😸

Reply:
I've rebased - the only difference should be that the fast_thread_switching fixture is gone as it was added by #11224.